modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -47,6 +48,12 @@ ifdef::ibm-cloud[]
 +
 The "Manually creating IAM for IBM Cloud" section of the installation content for IBM Cloud explains how to use the `ccoctl` utility to create new service IDs.
 endif::ibm-cloud[]
+--
++
+[NOTE]
+====
+If your cluster uses cluster capabilities to disable one or more optional components, delete the `CredentialsRequest` custom resources for any disabled components.
+====
```

Comment:
No Example `credrequests`?

Reply:
This covers all platforms, and the specifics will depend a bit on if they are using "normal" manual creds or manual creds with the `ccoctl` utility. They will need to refer our to the manually creating IAM content for their provider (as referenced in the previous step). This is a great thought though, so while I am not going to try to thread this needle on this PR, I am gonna think about it :nerd_face: